card/mod/05_standard/lib/file_uploader.rb

```diff
@@ -57,8 +57,8 @@ def filename
   def extension
     if file && file.extension.present?
       ".#{file.extension}"
-    elsif original_filename
-      File.extname(original_filename)
+#    elsif original_filename
+#      File.extname(original_filename)
```

Comment:
Is there a reason the above is necessary?  Those lookups are super expensive, and we do them many times (which is weird, too.  Why does this get called so many times??)